### PR TITLE
Fixes #71.

### LINKED ISSUE #71: Validate user sign up details
On signup : 
- Add checks for a valid and non-empty email id
- Add check for a valid and non-empty password
- Add a different field for first name & last name

Show proper error messages for each.

### PR DESCRIPTION
**Pull request by syedjahangirpeeran**

### CODE REVIEW

File: Android/Android.iml [hunk 1]
```diff
@@ -8,12 +8,12 @@
       </configuration>
```
Comment (Swati4star) [inline Android/Android.iml]: Remove this fie

**Comment by Swati4star** [comment]
This PR contains changes from other commits only.

**Comment by syedjahangirpeeran** [comment]
how to remove it??

File: Android/app/src/main/java/io/github/project_travel_mate/login/LoginActivity.java [hunk 2]
```diff
@@ -132,11 +138,28 @@ public void onClick(View view) {
...
                 String confirmPassString = confirm_pass_signup.getText().toString();
-                String nameString = name.getText().toString();
-                if (passString.equals(confirmPassString)) {
-                    mLoginPresenter.ok_signUp(nameString, emailString, passString, mHandler);
-                } else {
-                    Toast.makeText(this, R.string.passwords_check,
+                String firstname = firstName.getText().toString();
+                String lastname = lastName.getText().toString();
+                if (!emailString.equals("") && isValidEmail(emailString)){
+                    if (passString.length()>=8) {
```
Comment (Swati4star) [inline Android/app/src/main/java/io/github/project_travel_mate/login/LoginActivity.java]: The if-else loop looks quite complicated. Move it to a different function & simplify it.

File: Android/app/src/main/res/layout/activity_login.xml [hunk 3]
```diff
@@ -139,16 +139,32 @@
...
+                </android.support.design.widget.TextInputLayout>
+
                 <android.support.design.widget.TextInputLayout
                     android:layout_width="match_parent"
                     android:layout_height="wrap_content"
                     android:textColorHint="#ffffff">
 
                     <EditText
-                        android:id="@+id/input_name_signup"
+                        android:id="@+id/input_lastname_signup"
```
Comment (Swati4star) [inline Android/app/src/main/res/layout/activity_login.xml]: This shouldn't be a part of this fix

Comment (syedjahangirpeeran) [reply Android/app/src/main/res/layout/activity_login.xml]: In description it was given to add Last name field during sign up

**Pull request closed without merging**